tests/integration/test_update.py

```diff
@@ -184,6 +184,27 @@ def test_basic_local_update_direct_push(
     assert (remote_dir_clone / "README.packit").is_file()
 
 
+def test_update_downstream_changelog_even_if_has_autochangelog(
+    cwd_upstream,
+    api_instance,
+    distgit_with_autochangelog_and_remote,
+    mock_remote_functionality_downstream_autochangelog,
```

Comment:
Does this have any use in this test? If not, it could be removed.

Also, it would be nice to add a doc-string including a short description about what this test tries to do. This way readers can tell if the test code matches the intentions of the author :) Something like: "Check that a new entry is added to the %changelog section of the the spec-file in dist-git, when sync_changelog is set, even if the spec-file in dist-git uses %autochangelog".

Reply:
Yes, I need those lines to be able to have a distgit specfile with the %autochangelog macro.